csp-mixin/signals/gcpceOverview.libsonnet

```diff
@@ -0,0 +1,300 @@
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+            exprWrappers: [['topk(5,', ')']],
+          },
+        },
+      },
+
+      top5DiskWrite: {
+        name: 'Top 5 Instances by Disk write bytes',
+        description: 'List of top 5 instances by disk write bytes',
+        type: 'gauge',
+        aggFunction: 'sum',
+        unit: 'bytes',
+        sources: {
+          stackdriver: {
+            expr: 'increase(stackdriver_gce_instance_compute_googleapis_com_instance_disk_write_bytes_count{%(queriesSelector)s}[$__range])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+            exprWrappers: [['topk(5,', ')']],
+          },
+        },
+      },
+
+      top5DiskRead: {
+        name: 'Top 5 Instances by Disk read bytes',
+        description: 'List of top 5 instances by disk read bytes',
+        type: 'gauge',
+        aggFunction: 'sum',
+        unit: 'bytes',
+        sources: {
+          stackdriver: {
+            expr: 'increase(stackdriver_gce_instance_compute_googleapis_com_instance_disk_read_bytes_count{%(queriesSelector)s}[$__range])',
+            aggKeepLabels: ['instance_name'],
+            legendCustomTemplate: '',
+            exprWrappers: [['topk(5,', ')']],
+          },
+        },
+      },
+
+      // Table
+      tableCpuUtilization: {
+        name: 'tableCpuUtilization',
+        type: 'gauge',
+        aggFunction: 'sum',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s}',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+            exprWrappers: [['100 *', '']],
+          },
+        },
+      },
+      tableUptime: {
+        name: 'tableUptime',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_uptime_total{%(queriesSelector)s}',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableSentPackets: {
+        name: 'tableSentPackets',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_sent_packets_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableReceivedPackets: {
+        name: 'tableReceivedPackets',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_received_packets_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableSentBytes: {
```

Comment:
Sent bytes of what? maybe is more clear if you rename it as **tableNetworkSent**. Imagine if somebody else read the code. I had to read the metric to understand the purpose of this signal.